app/webpacker/components/MyCompetitions/index.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import {
+  Accordion,
+  Header,
+  Icon,
+  Checkbox, Segment,
+} from 'semantic-ui-react';
+import I18n from '../../lib/i18n';
+import {
+  personUrl,
+} from '../../lib/requests/routes.js.erb';
+import UpcomingCompetitionTable from './UpcomingCompetitionTable';
+import PastCompetitionsTable from './PastCompetitionTable';
+
+const defaultPermissions = {
+  can_attend_competitions: { scope: [] },
+  can_organize_competitions: { scope: [] },
+  can_administer_competitions: { scope: [] },
+};
+
+export default function MyCompetitions({ permissions, competitions, userInfo }) {
+  const [isAccordionOpen, setIsAccordionOpen] = useState(false);
+  const [shouldShowRegistrationStatus, setShouldShowRegistrationStatus] = useState(false);
```

Comment:
Bit of a side note, but recently I've started to like names like `accordionIsOpen` and `registrationStatusIsShown`. `is` is shorter than `should`, and putting it in the middle makes `if` statements more natural to read. Also, `should` kind of makes it sounds like it might not be, even though this is the source of truth.